### DISCUSSION TITLE
How does the GitHub community feel about personal achievement badges?

### DISCUSSION THREAD

**Discussion by bateman** [community/community#37346] [Other Feature Feedback, Questions, & Ideas]
Greetings,

I'm an academic researcher currently investigating how developers feel about personal achievement badges displayed on their GitHub profile pages.
I would appreciate it if you could help me by filling in this very short survey (~3-4 minutes): [https://forms.gle/Tfazu1a7Dw2XTJ7Z9](https://forms.gle/Tfazu1a7Dw2XTJ7Z9)

All the data will be anonymously stored and, once analyzed, it will be reported in aggregated form for research purposes only. After that, it will be deleted.

Thanks in advance for your help and consideration.

**Comment by radar-blip** [comment]
You're welcome.

**Reply by bateman** [reply]
Thanks @linuxlawson

**Comment by DSDmark** [comment]
Sure, I will do

**Reply by bateman** [reply]
Thank you! @DSDmark

**Comment by ossanna16** [comment]
Hi there @bateman and welcome to our community! Thank you for posting this interesting survey 🙂

**Reply by bateman** [reply]
Thanks for the feedback @ossanna16 !

**Reply by rustam201077** [reply]
yes, ok

**Reply by bateman** [reply]
hey @ossanna16 is there anyway you can help us spread further the word about this survey? We have too few data points so far. I tried with the official GitHub accout via Twitter but to no avail -- see [here](https://twitter.com/fcalefato/status/1585254840665772032?s=20&t=KKx4Jp8WNcaoBrNZi1-Ghg). 
Thanks in advance for your consideration.

**Reply by ossanna16** [reply]
Unfortunately I am unable to do this @bateman, sorry!

**Comment by rustam201077** [comment]
Да конечно

**Reply by bateman** [reply]
Thanks!

**Comment by xxtochoxx** [comment]
Sure, I will do

**Comment by lsy-73** [comment]
For a new developer, I think the badge is very important, it is an affirmation of one's own abilities and skills

**Reply by bateman** [reply]
Thanks for the feedback

**Comment by tinaponting** [comment]
Fun:) but not what I think of everytime I am in

**Comment by marcus-v-silva1** [comment]
ia fazer mais não dá mais
